src/web_peer_connection.cpp

```diff
@@ -126,6 +122,15 @@ web_peer_connection::web_peer_connection(peer_connection_args const& pack
 #endif
 }
 
+std::string web_peer_connection::escape_file_path(file_storage const& storage, file_index_t index)
```

Comment:
as far as I can tell, this does not need to be a member function. I think it should be a free function with internal linkage (anonymous namespace). and its declaration should be removed from the header